db/migrate/20220809130247_add_author_id_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddAuthorIdRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] The reference is not directly to user, it is instead author to table users. Kindly see the below example on how to make such migration for adding a reference with a different name than the reference model class name and make changes in your database. 🙏

![image](https://user-images.githubusercontent.com/77190203/183665204-85d7899a-ec56-4c94-9d6e-4325d1a89de4.png)

Reply:
Hi, @medaminedev66 Thank you for the review, I was wondering if I can update it directly in the file and then run db:migrate again or should I delete the database and create it again?

Reply:
Hi @EidHachem, yes you can update the migration directly, but do not forget to reset the database by running `rails db:reset`, then run `rails db:migrate`.